cli/source_maps.rs

```diff
@@ -6,8 +6,9 @@ use deno_core::error::JsError as CoreJsError;
 use sourcemap::SourceMap;
 use std::collections::HashMap;
 use std::str;
+use std::sync::Arc;
 
-pub trait SourceMapGetter {
+pub trait SourceMapGetter: Sync + Send {
```

Comment:
Why does it need to be Send + Sync?

Reply:
Because when I implemented it on `ProgramState`, `ProgramState` gets passed around as an `Arc` and therefore the trait needed to be `Sync + Send` to be able to be implementable in a thread safe way.